src/metpy/cbook.py

```diff
@@ -11,20 +11,20 @@
 
 from . import __version__
 
-pooch_kwargs = {'path': pooch.os_cache('metpy')}
+pooch_kwargs = {'path': pooch.os_cache('metpy'), 'version': 'v' + __version__}
 
 # Check if we have the data available directly from a git checkout, either from the
 # TEST_DATA_DIR variable, or looking relative to the path of this module's file. Use this
 # to override Pooch's path and disable downloading from GitHub.
 dev_data_path = os.environ.get('TEST_DATA_DIR', Path(__file__).parents[2] / 'staticdata')
 if Path(dev_data_path).exists():
     pooch_kwargs['path'] = dev_data_path
+    pooch_kwargs['version'] = None
     if pooch.__version__ >= 'v1.6.0':
         pooch_kwargs['allow_updates'] = False
 
 POOCH = pooch.create(
     base_url='https://github.com/Unidata/MetPy/raw/{version}/staticdata/',
-    version='v' + __version__,
     version_dev='main',
     **pooch_kwargs)
```

Comment:
Is there a use-case for "`TEST_DATA_DIR` exists but I still need to go fetch some data remotely?" One doesn't pop out to me immediately, I don't think.

Reply:
I'd say the only use case is a name collision with TEST_DATA_DIR existing in the environment since it's not prefixed with `METPY`. Otherwise, I'd say no.

The `version=None` though doesn't disable download, it just removes using the `version` to decide where to download from and where to put it.

Reply:
It's not disabled, but the URL isn't formatted if `version=None`

![image](https://user-images.githubusercontent.com/24628426/154335304-59c0249b-4ad9-4c27-8db9-a34ebf2ce48e.png)

Reply:
Crud. That makes sense. Ok, lemme think. The answer may be to also override URL.

Reply:
Ok, fixed by having the "dev" run manually fill in the URL.